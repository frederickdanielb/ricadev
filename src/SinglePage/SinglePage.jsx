const SinglePage = ({children}) => {
    return (
        <section className="single-section small-section p-0 pb-4 bg-white">
            {children}
        </section>
    );
}
export default SinglePage;