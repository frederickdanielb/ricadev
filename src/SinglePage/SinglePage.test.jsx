import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import SinglePage from "./SinglePage";
describe("Page Container component", () => {
	let component = null;
	beforeEach(() => {
		component = render(<SinglePage>Test</SinglePage>);
	});
	it("should render conrrectly", () => {
        expect(component).toBeTruthy()
        expect(component.asFragment()).toMatchSnapshot()
    });
	it("should render child", () => {
		expect(screen.getByText(/Test/)).toBeInTheDocument();
	});
});
