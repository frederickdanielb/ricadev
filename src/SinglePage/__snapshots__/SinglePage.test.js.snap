// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Container component should render conrrectly 1`] = `
<DocumentFragment>
  <section
    class="single-section small-section p-0 pb-4 bg-white"
  >
    Test
  </section>
</DocumentFragment>
`;
